frontend/catalyst/utils/jax_cpu_lapack_kernels/lapack_kernels_using_lapack.cc

```diff
@@ -0,0 +1,152 @@
+/* Copyright 2021 The JAX Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+/*
+ * This file is a modified version of
+ *
+ *   https://github.com/google/jax/blob/jaxlib-v0.4.28/jaxlib/cpu/lapack_kernels_using_lapack.cc
+ *
+ * from jaxlib-v0.4.28.
+ *
+ * See note in lapack_kernels.h for explanation of modifications.
+ */
+
+#include "lapack_kernels.h"
+
+// From a Python binary, JAX obtains its LAPACK/BLAS kernels from Scipy, but
+// a C++ user should link against LAPACK directly. This is needed when using
+// JAX-generated HLO from C++.
+
+extern "C" {
+
+jax::RealTrsm<float>::FnType cblas_strsm;
+jax::RealTrsm<double>::FnType cblas_dtrsm;
+jax::ComplexTrsm<std::complex<float>>::FnType cblas_ctrsm;
+jax::ComplexTrsm<std::complex<double>>::FnType cblas_ztrsm;
+
+jax::Getrf<float>::FnType LAPACKE_sgetrf;
+jax::Getrf<double>::FnType LAPACKE_dgetrf;
+jax::Getrf<std::complex<float>>::FnType LAPACKE_cgetrf;
+jax::Getrf<std::complex<double>>::FnType LAPACKE_zgetrf;
+
+jax::Geqrf<float>::FnType LAPACKE_sgeqrf;
```

Comment:
Could we add a comment at the beginning of each with what was modified?

Reply:
Ditto.